spec/services/migration/migrators/declaration_spec.rb

```diff
@@ -32,8 +32,7 @@ def setup_failure_state
         expect(declaration.state_reason).to eq(ecf_resource1.declaration_states.last.state_reason)
         expect(declaration.cohort.start_year).to eq(ecf_resource1.cohort.start_year)
         expect(declaration.lead_provider.ecf_id).to eq(ecf_resource1.cpd_lead_provider.npq_lead_provider.id)
-        expect(declaration.application.course.identifier.downcase).to eq(ecf_resource1.course_identifier.downcase)
```

Comment:
not sure why we removed those tests, they should still hold 🤔

Reply:
I thought they weren't need anymore as we're testing if application is correct. 
I've changed it back now.